analyzers/its/expected/Nancy/S3900-Nancy.Testing-net452.json

```diff
@@ -60,12 +60,6 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy.Testing/BrowserContextExtensions.cs#L104",
       "Location": "Line 104 Position 13-27"
     },
-    {
-      "Id": "S3900",
-      "Message": "Refactor this method to add validation of parameter \u0027cookies\u0027 before using it.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy.Testing/BrowserContextExtensions.cs#L119",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy.Testing/BrowserContextExtensions.cs#L119)
`NonNull` learnt in `!cookies.Any()` in the `if` condition.